pkg/test/resourcefixture/testdata/basic/accesscontextmanager/v1beta1/accesscontextmanagerserviceperimeter/dependencies.yaml

```diff
@@ -17,8 +17,6 @@ kind: Project
 metadata:
   annotations:
     cnrm.cloud.google.com/deletion-policy: "abandon"
-  labels:
```

Comment:
Just curious: why `metadata.labels` is removed in this test?

Reply:
I do not see any usage of project labels.